test/sms_nurse.test.js

```diff
@@ -58,6 +78,8 @@ describe("app", function() {
                     fixtures_ServiceRating().forEach(api.http.fixtures.add); // 150 - 169
                     fixtures_Jembi().forEach(api.http.fixtures.add);  // 170 - 179
                     fixtures_IdentityStore().forEach(api.http.fixtures.add); // 180 ->
+                    fixtures_Casepro().forEach(api.http.fixtures.add); // 242
```

Comment:
The comment at the end of this line indicates the fixtures that are being loaded. Since you added a fixture it's a good idea to update the comment.